cmd/cloud-controller-manager/gkenetworkparamsetcontroller.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	cloudprovider "k8s.io/cloud-provider"
+	networkclientset "k8s.io/cloud-provider-gcp/crd/client/network/clientset/versioned"
+	networkinformers "k8s.io/cloud-provider-gcp/crd/client/network/informers/externalversions"
+	gkenetworkparamsetcontroller "k8s.io/cloud-provider-gcp/pkg/controller/gkenetworkparamset"
+	"k8s.io/cloud-provider-gcp/providers/gce"
+	"k8s.io/cloud-provider/app"
+	cloudcontrollerconfig "k8s.io/cloud-provider/app/config"
+	genericcontrollermanager "k8s.io/controller-manager/app"
+	"k8s.io/controller-manager/controller"
+	kubecontroller "k8s.io/kubernetes/pkg/controller"
+)
+
+const JsonContentType = "application/json"
```

Comment:
```suggestion
const jsonContentType = "application/json"
```

do you need to export it?

Reply:
good catch, thanks